core-model/src/main/java/com/sequenceiq/cloudbreak/domain/ResourceStatus.java

```diff
@@ -1,6 +1,6 @@
 package com.sequenceiq.cloudbreak.domain;
 
-public enum NetworkStatus {
+public enum ResourceStatus {
```

Comment:
Maybe a different name would be better as it has nothing to do with any statuses